app.py

```diff
@@ -10,6 +11,34 @@ def hello_world():
 def agent_dumb():
 	return Response('{"type": 0, "args": {"target": {"party":0, "slot": 0}, "move": 0}}', mimetype="application/json")
 
+@app.route('/test')
```

Comment:
@dyc3 Is this supposed to be a GET request to http://localhost:4000/test ?

Reply:
yes